tests/test_gitlab.py

```diff
@@ -369,6 +369,22 @@ def test_initialization(self):
         self.assertEqual(gitlab.max_retries, MAX_RETRIES)
         self.assertEqual(gitlab.sleep_time, DEFAULT_SLEEP_TIME)
 
+        gitlab = GitLab('fdroid', 'fdroiddata', api_token='aaa', is_oauth_token=True, tag='test')
+
+        self.assertEqual(gitlab.owner, 'fdroid')
+        self.assertEqual(gitlab.repository, 'fdroiddata')
+        self.assertEqual(gitlab.origin, GITLAB_URL + '/fdroid/fdroiddata')
+        self.assertEqual(gitlab.tag, 'test')
+        self.assertIsNone(gitlab.client)
+        self.assertIsNone(gitlab.blacklist_ids)
+        self.assertEqual(gitlab.max_retries, MAX_RETRIES)
+        self.assertEqual(gitlab.sleep_time, DEFAULT_SLEEP_TIME)
+        self.assertEqual(gitlab.is_oauth_token, True)
+
+        with self.assertRaises(BackendError):
+            gitlab = GitLab('fdroid', 'fdroiddata', is_oauth_token=True, tag='test')
+
```

Comment:
Hi @jjmerchante. The unit tests passed but the flake8 test did n't. I think you can remove those extra lines (they seem to be the problem) so that all the tests pass. Thanks!

Reply:
Maybe it's better to move these two tests to a separate method. Furthermore if the variable `gitlab` is not used you can replace it with `_`, for instance:

```
with self.assertRaises(BackendError):
            _ = GitLab('fdroid', 'fdroiddata', is_oauth_token=True, tag='test')
```

A similar example can be found at: https://github.com/chaoss/grimoirelab-perceval/blob/master/tests/test_github.py#L1678